Frontend/voitheia-App/src/app/components/user/login/login.component.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../../variables.scss';
+
+
+#login_header {
+    color: black;
+    font-weight: 700;
+    text-align: center;
+    margin-bottom: 3rem;
+}
+
+
+.mat-input-element {
+    padding: 1.2rem;
+    font-size: 70%;
+    color: black;
+}
+
+
+#login_submit_btn {
+    margin: auto;
+    width: 50%;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  
+}
+#register_msg{
+    color: $gray-3;
+    margin: auto;
+    width: 100%;
+    text-align: center;
+
+}
+.hyperlink{
+    color: black;
+    text-decoration: underline;
+    cursor: pointer;
+}
+
+.hyperlink:hover{
+    color: $brand-color-dark;
```

Comment:
Same here: Hyperlinks will and should be handled in the style.scss that they are always the same

Reply:
moved to style.scss